core/include/msg.h

```diff
@@ -148,7 +281,7 @@ int msg_send_int(msg_t *m, kernel_pid_t target_pid);
  */
 static inline int msg_sent_by_int(const msg_t *m)
 {
-    return (m->sender_pid == KERNEL_PID_ISR);
+    return m->sender_pid == KERNEL_PID_ISR;
```

Comment:
"My" rule is basically always to have parens around any condition. Especially, I don't see any reason to remove them.